routers/web/repo/release.go

```diff
@@ -142,6 +142,11 @@ func releasesOrTags(ctx *context.Context, isTagList bool) {
 		return
 	}
 
+	if err = repo_model.InitReleaseRepo(ctx, releases...); err != nil {
+		ctx.ServerError("InitReleaseRepo", err)
+		return
+	}
```

Comment:
looks they are always in same repository, isn't it？

```suggestion
	for _, release := range releases {
		release.Repo = ctx.Repo.Repository
	}
```